src/createStore.test.ts

```diff
@@ -0,0 +1,54 @@
+import { compressToUTF16 } from 'lz-string';
+import { createStore } from './createStore';
+import { ISavedState, loadStoredState } from './localStorage';
+import { defaultState, IState } from './reducers/states';
+import { history } from './utils/history';
+
+const mockChangedStoredState: ISavedState = {
+  session: {
+    accessToken: 'yep',
+    refreshToken: 'refresherOrb',
+    role: undefined,
+    name: 'Jeff'
+  },
+  playgroundEditorValue: 'Nihao everybody',
+  playgroundIsEditorAutorun: true
+};
+
+const mockChangedState: IState = {
+  ...defaultState,
+  session: {
+    ...defaultState.session,
+    accessToken: 'yep',
+    refreshToken: 'refresherOrb',
+    role: undefined,
+    name: 'Jeff'
+  },
+  workspaces: {
+    ...defaultState.workspaces,
+    playground: {
+      ...defaultState.workspaces.playground,
+      editorValue: 'Nihao everybody',
+      isEditorAutorun: true
+    }
+  }
+};
+
+describe('CREATESTORE', () => {
+  test('has defaultState when initialised', () => {
+    expect(loadStoredState()).toEqual(undefined);
+    expect(createStore(history).getState()).toEqual({
+      ...defaultState,
+      router: { location: null }
+    });
+    expect(loadStoredState()).toEqual(undefined);
```

Comment:
What this expect line doing?

Reply:
Just to make sure it doesn't save state but yea probly don't need lol